full_name.rb

```diff
@@ -0,0 +1,15 @@
+# Write a program which asks for a person's first name, then middle, then last. It should store each of these parts in an array. Finally, it should greet the person using their full name. Call the program full_name.rb.
+
+def greet
+  names = []
+  %w[first middle last].each do |word|
+    puts "What is your #{word} name?"
+    user_input = gets.chomp
+    names << user_input
+  end
+  print 'Hello,'
+  names.each { |name| print " #{name}" }
+  puts '!'
+end
+
```

Comment:
I would suggest adding checks e.g if I input empty fields, the program could prompt me with "No empty fields!" or something similar. This is the current output if left as is.
<img width="238" alt="Screen Shot 2021-11-02 at 4 46 18 PM" src="https://user-images.githubusercontent.com/66847768/139949121-6fb49f0d-965f-45dc-b885-a77705e4ef33.png">